stacks-signer/src/config.rs

```diff
@@ -36,7 +36,7 @@ use wsts::state_machine::PublicKeys;
 /// List of key_ids for each signer_id
 pub type SignerKeyIds = HashMap<u32, Vec<u32>>;
 
-const EVENT_TIMEOUT_MS: u64 = 50;
+const EVENT_TIMEOUT_MS: u64 = 5000;
```

Comment:
One question is how this interacts with timeouts.  If our timeouts are on the order of minutes, then an extra 5sec wouldn't really affect things.  But if our timeouts are in single digit seconds, the 5sec blocking would alter the timing significantly, and not necessarily in the same way on different signers.